### DISCUSSION TITLE
so-feature-enable error

### DISCUSSION THREAD

**Discussion by kennysden** [Security-Onion-Solutions/securityonion#2695] [Unsupported Versions]
After running so-feature enable, I'm getting the error (Connection attempt timed out.  Unable to connect to ElasticSearch) when restarting the elastic script. Anyone got a fix to resolve this?

I read more into the error and this was the message 
![image](https://user-images.githubusercontent.com/59517722/105283558-bb38c980-5b7e-11eb-8ddb-20fb70dfc322.png)

**Comment by weslambert** [comment]
Could you still be using the old (OSS) Docker images/containers? We should be avoiding issues like this in the near future.

**Reply by kennysden** [reply]
not sure, how can I check? I'm used the 2.3.21 iso for installation

**Reply by weslambert** [reply]
You can run a command like `sudo docker images` to the see the version of images that exist on the box.

**Reply by kennysden** [reply]
yes 
![image](https://user-images.githubusercontent.com/59517722/105860003-23dad880-5fbb-11eb-9b03-daa3071db047.png)

**Comment by weslambert** [comment]
So, it looks like you'll need to adjust the outputs for Logstash to allow HTTP communication, by copying the various outputs for zeek/suricata/etc in `/opt/so/saltstack/default/salt/logstash/pipelines/config/so/` to the equivalent /opt/so/saltstack/local/salt/logstash/pipelines/config/so directory, modifying https -> http in the Elasticsearch output, then restart Logstash on each node with salt "*" cmd.run "so-logstash-restart". This issue should be resolved in the next resolve, but may be handled in a different manner at that time.
